### DISCUSSION TITLE
[Discussion] How to remove Bunkerweb completely?

### DISCUSSION THREAD

**Discussion by Myzel394** [bunkerity/bunkerweb#411] [Technical]
(This is more kind of a discussion, but I can't open a new discussions)

Here's the question:
How do I remove all associations to Bunkerweb completely in the frontend? I personally don't like to do this (as I'd love to support FOSS :D), but according to policy we need to remove all associations to Bunkerweb (basically everything that you can see in the frontend). I have already replaced 400, 401, 403, 404, 500, 501 and 502 error pages. I think the challenges page is missing, how do I replace that, and are there any more pages missing?

**Comment by TheophileDiot** [comment]
Hi, I'll convert this thread into a discussion
